python/sdk/prelude_sdk/controllers/build_controller.py

```diff
@@ -78,14 +78,17 @@ def undelete_test(self, test_id):
         raise Exception(res.text)
 
     @verify_credentials
-    def upload(self, test_id, filename, data):
+    def upload(self, test_id, filename, data, skip_compile=False):
         """ Upload a test or attachment """
         if len(data) > 1000000:
             raise ValueError(f'File size must be under 1MB ({filename})')
 
         h = self.account.headers | {'Content-Type': 'application/octet-stream'}
+        query_params = ''
+        if skip_compile:
+            query_params = '?skip_compile=true'
```

Comment:
i know we don't really use query params so this might be overkill but i usually use urllib.parse.encode(<param_dict>) to get the query param string. not super bothered either way though.

Reply:
since there's no user input in here, i think the end string will always be the exact same. but for the principle of it, i'll change :)